src/main/java/dev/galacticraft/mod/Constant.java

```diff
@@ -712,6 +712,7 @@ interface ScreenHandler {
         String OXYGEN_SEALER_SCREEN_HANDLER = "oxygen_sealer_screen_handler";
         String FUEL_LOADER_SCREEN_HANDLER = "fuel_loader_screen_handler";
         String AIR_LOCK_CONTROLLER_MENU = "air_lock_menu";
+        String NASA_WORKBENCH_SCREEN_HANDLER = "nasa_workbench_screen_handler";
```

Comment:
The name should end with menu since we are following mojmap standards and not yarn standards anymore